lib/net.js

```diff
@@ -492,7 +492,7 @@ Object.defineProperty(Socket.prototype, 'readyState', {
 
 
 Object.defineProperty(Socket.prototype, 'bufferSize', {
-  get: function() {
+  get: function() { // eslint-disable-line getter-return
```

Comment:
`eslint-disable-line getter-return` is unnecessary if this function returns `undefined` explicitly.

```js
  get: function() {
    if (this._handle) {
      return this[kLastWriteQueueSize] + this.writableLength;
    }
    return undefined;
  }
```